test/integration/MenuItem/MenuItem.spec.js

```diff
@@ -0,0 +1,15 @@
+import {assert} from 'chai';
+import React from 'react';
+import {shallow} from 'enzyme';
+import getMuiTheme from 'src/styles/getMuiTheme';
+import MenuItem from 'src/MenuItem/MenuItem';
+
+describe('<MenuItem />', () => {
```

Comment:
That's a unit test, not an integration test.
Could you move it here: `src/MenuItem/MenuItem.spec.js`?
Thanks